hybris/configure.ac

```diff
@@ -137,9 +137,9 @@ AC_ARG_WITH(default-egl-platform,
 AC_SUBST(DEFAULT_EGL_PLATFORM)
 
 if test "x$arch" = "xarm" -o "x$arch" = "xx86"; then
-  DEFAULT_HYBRIS_LD_LIBRARY_PATH="/vendor/lib:/system/lib"
+  DEFAULT_HYBRIS_LD_LIBRARY_PATH="/odm/lib:/vendor/lib:/system/lib"
```

Comment:
Here it is first...

Why?

Reply:
Go figure :D 

I picked the first precedent (from M)